autoload/asyncomplete.vim

```diff
@@ -276,7 +276,7 @@ function! s:remote_refresh(ctx, force) abort
         call asyncomplete#log('core', 's:remote_refresh', l:name, l:matchpos, a:ctx)
 
         let l:typed_len = l:endpos - l:startpos
-        if l:typed_len == 1
+        if !pumvisible() && l:typed_len >= g:asyncomplete_min_length
```

Comment:
1) The `pumvisible()` check seems a separate issue from this PR
2) I _think_ `==` should work in this context, don't need `>=`.